modules/core/src/main/java/org/apache/ignite/internal/commandline/CommonArgParser.java

```diff
@@ -336,4 +369,70 @@ private String securityWarningMessage(String password) {
 
         return String.format(pwdArgWarnFmt, password, password);
     }
+
+    /**
+     * Extracts user properties from property string or file.
+     *
+     * @param userPropStr {@code String} Property string.
+     * @param userPropPath {@code String} Property file path.
+     */
+    private Map<String, String> userProperties(String userPropStr, String userPropPath){
```

Comment:
Space between `)` and `{` in the end of line.

Reply:
fixed